decoders/connector/daytec/siglog_32a2/v1.0.0/payload-config.jsonc

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "../../../../../schema/connector_details.json",
+  "description": "../description.md",
+  "install_text": "**Connect your Siglog 32A2 through your network.**\n\nSiglog 32A2 is a wireless transmitter, used to monitor 32 digital inputs and 2 analog inputs\n\n\n>",
+  "install_end_text": "Device created sucessful.\n\nYou can get a Siglog Dashboard template [here.](https://admin.tago.io/template/6613d2cc594ba70010abbd05)\n\nJust add to your Dashboards and select the device you just created. \n\nFor more information about our products, [Contact Us](http://www.day-tec.com).",
```

Comment:
The device creation already states when a device if a device was created, Please remove "Device created sucessful."

Also, I opened the template and noticed that the device name is "0015-open"
![image](https://github.com/user-attachments/assets/9ef33b39-da31-4161-ad8e-1c40773a91ed)

Can you change this to Siglog 32A2?

Also we no longer allow links to external sites since they can broken later on.